themes/bootstrap3/js/resultcount.js

```diff
@@ -2,19 +2,29 @@
 
 VuFind.register('resultcount', function resultCount() {
   function init() {
-    $('ul.nav-tabs [data-show-counts] a').each(function queryResultCount(){
-      var $this = $(this);
-      if ($this.attr('href') !== undefined) {
-        var queryString = $this.attr('href');
-        var source = $this.data('source');
-        $.ajax({
-          url: VuFind.path + '/AJAX/JSON?method=getResultCount',
-          dataType: 'json',
-          data: {querystring: queryString, source: source},
-          success: function appendResultCount(response){
-            $this.append(' (' + response.data.total.toLocaleString() + ')');
+    document.querySelectorAll('ul.nav-tabs [data-show-counts] a').forEach((tab) => {
+      function loadCount(url) {
+        let source = tab.dataset.source;
+        let xhttp = new XMLHttpRequest();
+        xhttp.onreadystatechange = function appendResultCount() {
+          if (this.readyState === 4 && this.status === 200) {
+            let response = JSON.parse(xhttp.responseText);
+            tab.textContent += ' (' + response.data.total.toLocaleString() + ')';
           }
+        };
+        let params = new URLSearchParams({
+          method: 'getResultCount',
+          querystring: url,
+          source: source
         });
+        xhttp.open('GET', VuFind.path + '/AJAX/JSON?' + params.toString());
+        xhttp.setRequestHeader('Content-type', 'application/json');
+        xhttp.send();
+      }
+      if (tab.getAttribute('href')) {
```

Comment:
Could this be simplified to `loadCount(tab.getAttribute('href') || tab.dataset.searchUrl)` (maybe in combination with an early return from loadCount if a falsy value is passed in)?

Reply:
Done